pyuvdata/uvdata/tests/test_uvdata.py

```diff
@@ -6792,3 +6793,123 @@ def test_multifile_read_errors(read_func, filelist):
         "Reading multiple files from class specific read functions is no "
         "longer supported."
     )
+
+
+def test_multifile_read_check():
+    """Test setting check_file_status=True when reading in files"""
+
+    uv = UVData()
+    uvh5_file = os.path.join(DATA_PATH, "zen.2458661.23480.HH.uvh5")
+
+    # Create a test file and remove header info to 'corrupt' it
+    testfile = os.path.join(DATA_PATH, "test", "zen.2458661.23480.HH.uvh5")
+    uv.read(uvh5_file)
+    uv.write_uvh5(testfile)
+    with h5py.File(testfile, "r+") as h5f:
+        del h5f["Header/ant_1_array"]
+
+    # Test that the expected error arises
+    with pytest.raises(KeyError) as cm:
+        uv.read(testfile, skip_bad_files=False)
+    assert "Unable to open object (object 'ant_1_array' doesn't exist)" in str(cm.value)
```

Comment:
you don't need to change this, but FYI `pytest.raises` also can take the `match` parameter in the same way that `pytest.warns` does, so you don't have to write a separate assert.